INSTALL.md

```diff
@@ -164,6 +164,18 @@ Then you can try one of the
 [examples](https://github.com/kubernetes/client-go/tree/v2.0.0/examples/) from
 the 2.0.0 release.
 
+For client-go 3.0.0 must be defined `k8s.io/apimachinery` override in Gopkg.toml
+
+```
+[[constraint]]
+  name = "k8s.io/client-go"
+  version = "3.0.0"
+
+[[override]]
```

Comment:
`constraint` don't work... Transitive dependency are master (because apimachinery doesn't versioned)... Master don't work with version 3.0.0... Only one way is `override` more about this in #256